src/main/java/io/odpf/firehose/config/S3Config.java

```diff
@@ -0,0 +1,21 @@
+package io.odpf.firehose.config;
+
+import org.aeonbits.owner.Config;
+
+public interface S3Config extends Config {
+    @Key("S3_REGION")
+    String getS3Region();
+
+    @Key("S3_BUCKET_NAME")
+    String getS3BucketName();
+
+    @Key("${GCS_TYPE}_S3_ACCESS_KEY")
+    String getS3AccessKey();
+
+    @Key("${GCS_TYPE}_S3_SECRET_KEY")
```

Comment:
GCS_TYPE prefix is not correct for S3

Reply:
@lavkesh @ravisuhag should we maintain  same structure similar to GCS in S3 as well . For example **${S3_TYPE}_S3_ACCESS_KEY**

Reply:
yea, we should, it can also be used as DLQ then.

Reply:
also you might need to check dlqwriter code.